README.md

```diff
@@ -5,18 +5,18 @@ This is practice tool for Paper Mario 64 intended for use on N64 with an Everdri
 The latest release and installation instructions can be found on the [releases page](https://github.com/jcog/fp/releases).
 
 # Usage
-See the [user manual](https://github.com/JCog/fp/blob/master/USAGE.md) for information on how to use fp.
+See the [user manual](https://github.com/JCog/fp/blob/develop/USAGE.md) for information on how to use fp.
```

Comment:
this should remain as master. until we have the manual on practicerom.com, we want to direct people toward the version for the latest release, which will always be master.